experimental/python/codegen/pyproject.toml

```diff
@@ -0,0 +1,12 @@
+[project]
+name = "codegen"
+version = "0.1.0"
+description = "Code generator for Python"
+requires-python = ">=3.10"
```

Comment:
The Python version can be specified either here or in `.python-version`.

If there is a specific need for 3.13 it would be nice into include a comment saying why it is required.